src/libraries/Pausable.sol

```diff
@@ -105,4 +160,28 @@ abstract contract Pausable is Initializable {
         PauseStorage storage $ = _getPauseStorage();
         return $._pauseFlag == PAUSED;
     }
+
+    /**
+     * @dev Transfers the ability to pause to a new account (`newPauser`).
+     */
+    function _transferPauserCapability(address newPauser) internal virtual onlyPauser {
+        PauserStorage storage $ = _getPauserStorage();
+        address oldPauser = $._pauser;
+        $._pauser = newPauser;
+        emit PauserTransferred(oldPauser, newPauser);
+    }
+
+    /**
+     * @dev Leaves the contract without a Pauser
+     * and all the capabilities afforded with that role.
+     * TODO: do we need this?
+     */
+    function renouncePauser() public virtual onlyPauser {
```

Comment:
If the owner has the ability to update the pauser then I'm fine with keeping this in. This PR will need some changes to the inheriting contracts

Reply:
Agreed, if owner has ability to update pauser we should be good to keep this.